content/ratatui-0.23.0.md

```diff
@@ -0,0 +1,276 @@
+
+Such an adventure we had! And the best part is, it continues... In that regard, I would especially like to thank [Florian Dehau](https://github.com/fdehau) since he created `tui-rs` and made all of this possible! 💖
+
+Now, let's get to what's new in `ratatui`! Buckle up, we're about to cook up some terminal interfaces.
+
+**Full changelog**: [https://github.com/ratatui-org/ratatui/releases/tag/v0.23.0](https://github.com/ratatui-org/ratatui/releases/tag/v0.23.0)
+
+---
+
+### Coolify everything 😎
+
+We already had a cool name and a logo, and now we have a cool description as well:
+
+```diff
+- ratatui: A Rust library to build rich terminal user interfaces or dashboards.
++ ratatui: A Rust library that's all about cooking up terminal user interfaces.
+```
+
+We also renamed our organization from `tui-rs-revival` to `ratatui-org`:
+
+- [**https://github.com/ratatui-org/ratatui**](https://github.com/ratatui-org/ratatui)
+
+---
+
+### Barchart: horizontal bars
+
+You can now render the bars horizontally for the `Barchart` widget. This is especially useful in some cases to make more efficient use of the available space.
+
+Simply use the `Direction` attribute for rendering horizontal bars:
+
+```rs
+let mut barchart = BarChart::default()
+    .block(Block::default().title("Data1").borders(Borders::ALL))
+    .bar_width(1)
+    .group_gap(1)
+    .bar_gap(0)
+    .direction(Direction::Horizontal);
+```
+
+Here is an example of what you can do with the `Barchart` widget (see the bottom right for horizontal bars):
+
+![horizontal bars](https://user-images.githubusercontent.com/15311024/263077132-ff80b2c4-9d2d-480b-9a72-f3779688bec8.png)
+
+---
+
+### Voluntary skipping capability for Sixel
+
+> [Sixel](https://en.wikipedia.org/wiki/Sixel) is a bitmap graphics format supported by terminals.
+> "Sixel mode" is entered by sending the sequence `ESC+Pq`.
+> The "String Terminator" sequence `ESC+\` exits the mode.
+
+`Cell` widget now has a `set_skip` method that allows the cell to be skipped when copying (diffing) the buffer to the screen. This is helpful when it is necessary to prevent the buffer from overwriting a cell that is covered by an image from some terminal graphics protocol such as Sixel, iTerm, Kitty, etc.
+
+See the pull request for more information: [https://github.com/ratatui-org/ratatui/pull/215](https://github.com/ratatui-org/ratatui/pull/215)
+
+In this context, there is also an experimental image rendering crate: [_ratatu-image_](https://github.com/benjajaja/ratatu-image)
+
+![ratatu-image](https://raw.githubusercontent.com/benjajaja/ratatu-image/master/assets/Recording.gif)
+
+---
+
+### Table/List: Highlight spacing
+
+We added a new property called `HightlightSpacing` to the `Table` and `List` widgets and it can be optionally set via calling `hightlight_spacing` function.
```

Comment:
```suggestion
We added a new property called `HighlightSpacing` to the `Table` and `List` widgets and it can be optionally set via calling `highlight_spacing` function.
```

Reply:
Good catch, thanks!